pyo3_pack/__init__.py

```diff
@@ -100,6 +100,16 @@ def get_requires_for_build_sdist(config_settings=None):
 
 # noinspection PyUnusedLocal
 def prepare_metadata_for_build_wheel(metadata_directory, config_settings=None):
+    print("Checking for Rust toolchain....")
+    output = subprocess.check_output(["cargo", "--version"])
+    if not "cargo" in output:
```

Comment:
Running this locally in a venv with python 3.6 on linux I get a `TypeError: a bytes-like object is required, not 'str'`

Reply:
Oh, yes, it's back to `.decode("utf-8", "ignore")`.